src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static Connection getConnection() {
+        try {
+            Properties dbProperties = new Properties();
+            dbProperties.put("user", "root");
```

Comment:
The best way would be to externalise properties in config.

But here we can at least move it to class level constants

private static final String USER_PROPERTY = "user";
private static final String PASSWORD_PROPERTY = "password";

And values also 

private static final String USER_VALUE = "root";
private static final String PASSWORD_VALUE = "1234567";